libs/barista-components/schematics/src/ng-add/__snapshots__/ng-add.spec.ts.snap

```diff
@@ -468,12 +467,12 @@ Object {
     "@angular/animations": "^8.2.12",
     "@angular/cdk": "^8.2.12",
     "@angular/common": "^8.2.12",
-    "@angular/compiler": "~10.1.6",
+    "@angular/compiler": "~11.0.5",
```

Comment:
We should use ^11.0.5 here and for all other angular related dependencies as well. Angular follows semver so it it safe to have a rolling minor version.